cmd/admission-controller/main.go

```diff
@@ -19,13 +19,13 @@ import (
 	"os/signal"
 	"syscall"
 
-	"github.com/golang/glog"
+	glog "k8s.io/klog"
```

Comment:
all k8s code switch to use `k8s.io/klog` now, alias can be removed in the future